src/Template/ModelTemplate.cs

```diff
@@ -228,7 +232,19 @@ public void Write(ClientModel model, JavaFile javaFile)
                             comment.Param(property.Name, $"the {property.Name} value to set");
                             comment.Return($"the {model.Name} object itself.");
                         });
-                        classBlock.PublicMethod($"{model.Name} {property.SetterName}({propertyClientType} {property.Name})", (methodBlock) =>
+                        string parameterType = propertyClientType.ToString();
+                        if (property.UseVarArg)
+                        {
+                            if (!(property.WireType is ListType))
+                            {
+                                throw new InvalidOperationException($"Cannot create a varArgs for a non ListType '{property.WireType}'");
+                            }
+                            if (string.IsNullOrWhiteSpace(property.XmlListElementName)) {
+                                throw new InvalidOperationException($"Could not determine the varArgs element");
+                            }
+                            parameterType = $"{property.XmlListElementName}...";
```

Comment:
just for my understanding, what are the three dots for? :)

Reply:
That's the [varargs](https://docs.oracle.com/javase/8/docs/technotes/guides/language/varargs.html) syntax :)